src/terms.js

```diff
@@ -248,6 +248,7 @@ export default [
     'real.time',
     'real.world',
     'reconfigure',
+    'redefin(e|(ing)?)',
```

Comment:
```suggestion
    'redefin(e|ing)',
```

No need for the parens there I think (because we don't need to capture "redefin")